client/components/wcc-settings-form/index.js

```diff
@@ -35,7 +35,10 @@ const WCCSettingsForm = ( props ) => {
 		const label = ( currentStepLayout || {} ).action_label || __( 'Next' );
 		return (
 			<FormButtonsBar>
-				<FormButton type="button" onClick={ props.formActions.nextStep }>
+				<FormButton
+					type="button"
+					disabled={ ! currentStepLayout || 0 < props.stepErrors.length || props.form.isSaving }
+					onClick={ props.formActions.nextStep }>
```

Comment:
The single-step form doesn't have any logic here. If you look at the `render` method, if the form is single-step, then it just render a `<WCCSettingsStep>` component. So it's very unlikely that all this logic would break the single-step forms (famous last words)